packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelSummary.tsx

```diff
@@ -126,7 +126,7 @@ function TotalGasFees() {
    */
   return (
     <>
-      {isDepositMode && (
+      {!isWithdrawalMode && (
```

Comment:
`isDepositOrTeleportMode`?

Reply:
imo we should use `isDepositOrTeleportMode` instead of `!isWithdrawalMode` everywhere, it's easier to follow that it's for deposits and teleports